content/docs/next-release/getting-started.md

```diff
@@ -8,7 +8,9 @@ If you are new to distributed tracing, please check the [Introduction](../) page
 
 ## Instrumentation
 
-Your applications must be instrumented before they can send tracing data to Jaeger backend. Check the [Client Libraries](../client-libraries) section for information about how to use the OpenTracing API and how to initialize and configure Jaeger tracers.
+Your applications must be instrumented before they can send tracing data to Jaeger. We recommend using the [OpenTelemetry](https://opentelemetry.io) instrumentation and SDKs.
```

Comment:
I'm not sure if "We recommend using the OpenTelemetry SDKs for instrumentation" reads better or not?

Reply:
I felt in this context the focus is more on the instrumentation (cf. previous sentence).